python/dask_cudf/record.txt

```diff
@@ -1,65 +1,65 @@
-/conda/envs/gdf/lib/python3.7/site-packages/dask_cudf/core.py
```

Comment:
Not sure why `record.txt` is in the repo -- @kkraus14 should this file be added to .gitignore ?

Reply:
Yes it should be.

Reply:
Ok, I will delete and ignore it